testing/tests/DevExpress.ui.widgets.dataGrid/editing.tests.js

```diff
@@ -12161,7 +12161,8 @@ QUnit.module('Editing with validation', {
             }
         });
 
-        rowsView.scrollChanged.add(function() {
+        const scrollHandler = function() {
+            rowsView.scrollChanged.remove(scrollHandler);
```

Comment:
Test was imported from 17_1, we will reinvestigate this scenario if there will be new issue

Reply:
Yes, I agree with your concept